plugins/woocommerce-admin/client/marketing/overview-multichannel/MarketingOverviewMultichannel.tsx

```diff
@@ -7,21 +7,52 @@ import { useUser } from '@woocommerce/data';
  * Internal dependencies
  */
 import { getAdminSetting } from '~/utils/admin-settings';
+import { Channels } from './Channels';
 import { InstalledExtensions } from './InstalledExtensions';
 import { DiscoverTools } from './DiscoverTools';
 import { LearnMarketing } from './LearnMarketing';
 import '~/marketing/data';
+import '~/marketing/data-multichannel';
+import {
+	useRegisteredChannels,
+	useRecommendedChannels,
+} from '~/marketing/hooks';
 import './MarketingOverviewMultichannel.scss';
+import { CenteredSpinner } from '../components';
 
 export const MarketingOverviewMultichannel: React.FC = () => {
+	const {
+		loading: loadingRegistered,
+		data: dataRegistered,
+		refetch,
+	} = useRegisteredChannels();
+	const { loading: loadingRecommended, data: dataRecommended } =
+		useRecommendedChannels();
 	const { currentUserCan } = useUser();
 
 	const shouldShowExtensions =
 		getAdminSetting( 'allowMarketplaceSuggestions', false ) &&
 		currentUserCan( 'install_plugins' );
 
+	if (
+		( loadingRegistered && ! dataRegistered ) ||
+		( loadingRecommended && ! dataRecommended )
```

Comment:
I guess we can omit here `! dataRegistered` and `! dataRecommended` and rely only on the loading states for the loading spinner.

Reply:
Gotcha thanks